Composer/plugins/vacore/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "vacore",
+  "version": "1.0.0",
+  "description": "provide functionality around using VA core",
```

Comment:
Nit: Provides
How about:  A plug-in to provide Virtual Assistant creation ? (or something like that)